### PR TITLE
test Update README.md

### PR DESCRIPTION
**Pull request by gandalf-repo**
every pr needs a desc

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,6 +11,3 @@
 | 6 | Merged PR shows up | Merge PR |  |  |
 | 7 | Build shows up | Build commits show up | Build pipeline triggers |  |   
 | 8 | Status: Done |  | Build completes |  |
```
Comment (arvi18) [inline README.md:13]: why this is being removed ?

Comment (arvi18) [inline README.md:13]: can we improve this table?

Comment (gandalf-repo) [reply README.md:13]: put a flowchart instead of this

File: __tests__/index.test.js [hunk 2]
```diff
@@ -6,7 +6,7 @@ const add = require("../index");
 test("adds 1 + 2 to equal 3", () => {
   expect(add(1, 2)).toBe(3);
 });
-
+//
 // this might fail randomly
 test("might fail randomly", () => {
   expect(add(1, 2)).toBe(Math.random() > 0.5 ? 3 : 4);
```
Comment (arvi18) [inline __tests__/index.test.js:12]: This is a very log comment.
Not expected but lets see how this affects the message to be sent on slack.
This is a very log comment.
Not expected but lets see how this affects the message to be sent on slack.
This is a very log comment.
Not expected but lets see how this affects the message to be sent on slack.

File: README.md [hunk 3]
```diff
@@ -1,5 +1,6 @@
 # hercules-tools
 
+
 | Step | JIRA | GitHub | Jenkins | PagerDuty |
```
Comment (arvi18) [inline README.md:4]: starting a review

File: README.md [hunk 4]
```diff
@@ -10,6 +11,3 @@
 | 6 | Merged PR shows up | Merge PR |  |  |
 | 7 | Build shows up | Build commits show up | Build pipeline triggers |  |
```
Comment (arvi18) [inline README.md:12]: review start 2

File: __tests__/index.test.js [hunk 5]
```diff
@@ -6,7 +6,7 @@ const add = require("../index");
 test("adds 1 + 2 to equal 3", () => {
   expect(add(1, 2)).toBe(3);
 });
```
Comment (arvi18) [inline __tests__/index.test.js:8]: review 3

**Review by arvi18 — COMMENTED** [review]
reviewed

**Pull request merged**